### ISSUE TITLE
Add Ledger support on mobile

### ISSUE THREAD

**Issue by b70v** [greymass/anchor#1178]
There is currently no way to manage EOS with a ledger device on mobile devices. If Anchor added Ledger support for mobile by Bluetooth for the Nano X and wired on Android for the Nano S it would be really helpful.

**Comment by aaroncox** [comment]
I completely agree that this is needed. It's something that's been on our radar for a while now - but completing this will require a significant amount of effort, which we just haven't been able to allocate with our small team yet. Hopefully we'll be able to coordinate the development of this as our team grows and we knock out some of the more urgent tasks from our list.
